src/controller/expenses.controller.js

```diff
@@ -0,0 +1,107 @@
+const expensesService = require('../services/expenses.service');
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  const { userId, from, to, categories } = req.query;
+
+  if (!userId && !categories && (!from || !to)) {
+    res.send(expensesService.getAllExpenses());
+  }
```

Comment:
You have a potential bug here. If the user doesn't provide any parameters in the query, your API will return all expenses. It might be a security issue if you don't want to expose all the data to everyone. Also, it might lead to performance issues if the number of expenses is large. Consider adding a default behavior, like returning an error message or a limited number of expenses.